datasets/finer/finer.py

```diff
@@ -0,0 +1,186 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+""""The Finnish News Corpus for Named Entity Recognition dataset."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import logging
+
+import datasets
+
+
+_CITATION = """\
+@article{ruokolainen2019finnish,
+  title={A finnish news corpus for named entity recognition},
+  author={Ruokolainen, Teemu and Kauppinen, Pekka and Silfverberg, Miikka and Lind{\'e}n, Krister},
+  journal={Language Resources and Evaluation},
+  pages={1--26},
+  year={2019},
+  publisher={Springer}
+}
+"""
+
+_DESCRIPTION = """\
+The directory data contains a corpus of Finnish technology related news articles with a manually prepared
+named entity annotation (digitoday.2014.csv). The text material was extracted from the archives of Digitoday,
+a Finnish online technology news source (www.digitoday.fi). The corpus consists of 953 articles
+(193,742 word tokens) with six named entity classes (organization, location, person, product, event, and date).
+The corpus is available for research purposes and can be readily used for development of NER systems for Finnish.
+"""
+
+_URLS = {
+    "train": "https://github.com/mpsilfve/finer-data/raw/master/data/digitoday.2014.train.csv",
+    "dev": "https://github.com/mpsilfve/finer-data/raw/master/data/digitoday.2014.dev.csv",
+    "test": "https://github.com/mpsilfve/finer-data/raw/master/data/digitoday.2015.test.csv",
+    "test_wikipedia": "https://github.com/mpsilfve/finer-data/raw/master/data/wikipedia.test.csv",
+}
+
+
+class FiNERConfig(datasets.BuilderConfig):
+    """BuilderConfig for FiNER dazsez."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for FiNER dataset.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(FiNERConfig, self).__init__(**kwargs)
+
+
+class FiNER(datasets.GeneratorBasedBuilder):
```

Comment:
Could you rename it Finer ? We're using camel case for class names.
Same for the config